plugins/dell/meson.build

```diff
@@ -1,5 +1,9 @@
 cargs = ['-DG_LOG_DOMAIN="FuPluginDell"']
 
+install_data(['dell.quirk'],
+  install_dir: join_paths(get_option('datadir'), 'fwupd', 'quirks.d')
+)
+
```

Comment:
If those other strings can't work, would pre-processing (.quirks.in ->.quirks) make sense rather than hardcoding in source in two places?